assets/css/dogs-cats-birds-rabbits.css

```diff
@@ -1,51 +1,49 @@
 
 .dogs-cats-birds-rabbits-wrapper{
     height:100vh;
-    width:100%;
     display:flex;
-    background:grey;
-    position:relative;
-}
-.dogs-cats-birds-rabbits-wrapper::after{
-    position:absolute;
-    content:' ';
-    height:100%;
-    width:10%;
-    background:grey;
-    left:-10%;
-}
-.dogs-cats-birds-rabbits-wrapper::before{
-    position:absolute;
-    content:' ';
-    height:100%;
-    width:10%;
-    background:grey;
-    right:-10%;
-}
-.inner-wrapper-small-margin{
-    width:85%;
+    background:#e5e5e5;
+    width:100%;
     margin:0 auto;
 }
 .dogs-cats-birds-rabbits-photo{
    width:40%;
    height:80%;
-   background:blue;
    position:relative;
    top:10%;
 }
-.dogs-cats-birds-rabbits-rectangle-wrapper{
+.dogs-cats-birds-rabbits-box-wrapper{
     width:60%;
     height:100%;
```

Comment:
Height nie działa w ten sposób. To nie jest position absolute. Jeśli to byłby absolute i pozycjonowałbyś go względem rodzica relative Z OKREŚLONĄ wysokością, to wtedy ustawiłby 100% wysokości względem tego rodzica.

Teraz ustawiasz 100% czego? Skąd on ma to wyliczyć? To wyrzucenia

Reply:
Poprawione